src/backend/distributed/master/master_metadata_utility.c

```diff
@@ -200,6 +201,49 @@ ShardLength(uint64 shardId)
 
 
 /*
+ * NodeHasActiveShardPlacements returns whether any active shards are placed on this node
+ * TODO: For each shard in this node, check the existence of any active placements on any
+ * other node so.
+ * TODO: Re-replicate remaining placements.
+ * TODO: Remove relevant rows from pg_dist_shard_placement automatically
```

Comment:
Shoul I move them to header comments of master_add_node or completely remove them?